src/lib/layer-manager.js

```diff
@@ -77,7 +80,18 @@ export default class LayerManager {
         layerId: null
       }
     };
+
     Object.seal(this.context);
+
+    seer.listenFor('deck.gl', payload => {
```

Comment:
Just like in luma.gl, I am not suggesting we rename `seer` to `seer-integration`. 

Just like in luma.gl, I am suggesting we collect the new seer integration code here inside deck.gl into a new file `seer-integration.js`. 

That file would export a couple of functions that we call in the layer-manager and layer etc.

The purpose is to avoid adding long `seer`-related blocks of code to other classes. If possible, the code in those classes should focus on the main purpose of those classes, not on instrumentation.